### ISSUE TITLE
Some messages in dark theme are hard to read

### ISSUE THREAD

**Issue by carun** [vector-im/element-web#9366]
### Description

Some messages in dark theme are hard to read. The dark theme needs tweaking.

### Steps to reproduce

- Connect to IRC channel, like `#mercurial`

Describe how what happens differs from what you expected.

Anytime a notification comes, the font is hard to read. Check the item marked "red" in the below screenshot. The font should be readable for various kinds of messages.

Log: not sent    

![image](https://user-images.githubusercontent.com/471374/55436635-af1aec80-5551-11e9-8c02-15e43aa3145d.png)

### Version information

- **Platform**: web (in-browser)

For the web app:

- **Browser**: Chrome 
- **OS**: Windows 10
- **URL**: riot.im/app

**Comment by turt2live** [comment]
This is technically a duplicate of https://github.com/vector-im/riot-web/issues/2988#issuecomment-318893316, however that is a comment and not a dedicated issue. Would probably be fixed by https://github.com/vector-im/riot-web/issues/5424, although ultimately there's not much we can do here: the sender is saying it should be dark blue, and Riot is respecting that.

**Comment by Palid** [comment]
Closing in favor of https://github.com/vector-im/element-web/issues/5424
